SourceStream/file/checksum.py

```diff
@@ -0,0 +1,37 @@
+from bs4 import BeautifulSoup as bs
+import requests
+
+
+def checksum(): 
+    # gets page contents
+    request = requests.get('https://linuxfromscratch.org/lfs/view/stable-systemd/chapter03/packages.html', headers={'User-Agent': 'Mozilla/5.0'})
```

Comment:
After I've finished version 1.0 of SplashOS I don't wanna be using what LFS suggests and do my own thing.